### PR TITLE
Feature cassandra version upgrade

### PR DESCRIPTION
**Pull request by lithinwxd**
## Description
SSL enablement for Cassandra Connector 

## Motivation and Context
we were not able to connect to Cassandra via SSL . We were getting the below exception 
com.datastax.driver.core.exceptions.NoHostAvailableException: All host(s) tried for query failed (tried: /52.116.130.195:9042 (com.datastax.driver.core.exceptions.OperationTimedOutException: [/52.116.130.195:9042] Operation timed out))

It is using 3.6.0 version of DataStax driver which does not support SSL. Version 3.11.2 or higher version of DataStax driver supports SSL.

## Contributor checklist

- [ ] Please make sure your submission complies with our [development](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#development), [formatting](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#formatting), [commit message](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#commit-formatting-and-pull-requests), and [attribution guidelines](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#attribution).
- [ ] PR description addresses the issue accurately and concisely.  If the change is non-trivial, a GitHub Issue is referenced.
- [ ] Documented new properties (with its default value), SQL syntax, functions, or other functionality.
- [ ] If release notes are required, they follow the [release notes guidelines](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines).
- [ ] Adequate tests were added if applicable.
- [ ] CI passed.

## Release Notes
Please follow [release notes guidelines](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines) and fill in the release notes below.

```
== RELEASE NOTES ==

General Changes
* ... :pr:`12345`
* ... :pr:`12345`

Hive Connector Changes
* ... :pr:`12345`
* ... :pr:`12345`
```

If release note is NOT required, use:

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

File: presto-cassandra/pom.xml [hunk 1]
```diff
@@ -17,8 +17,9 @@
 
     <dependencies>
         <dependency>
-            <groupId>com.facebook.presto.cassandra</groupId>
-            <artifactId>cassandra-driver</artifactId>
+            <groupId>com.datastax.cassandra</groupId>
+            <artifactId>cassandra-driver-core</artifactId>
```
Comment (elbinpallimalilibm) [inline presto-cassandra/pom.xml:21]: Is this a drop in replacement for the presto jar? Are both shaded jars?

Comment (lithinwxd) [reply presto-cassandra/pom.xml:21]: This is not shaded jar , this is actual datastax jar . cassandra-driver-core internally was using datastax 3.6.0 and no further update was done . so we removed that dependancy and used the datastax jar directly

Comment (lithinwxd) [reply presto-cassandra/pom.xml:21]: Presto is using the below presto-cassandra driver for Cassandra https://mvnrepository.com/artifact/com.facebook.presto.cassandra/cassandra-driver/3.6.0-1

This is a shaded jar that has DataStax driver bundled inside

com.datastax.cassandra cassandra-driver-core 3.6.0 runtime
It is using 3.6.0 version of DataStax driver which does not support SSL. Version 3.11.2 of DataStax driver supports SSL.

Comment (elbinpallimalilibm) [reply presto-cassandra/pom.xml:21]: Have you tested this version with the community version of Presto itself?

Comment (lithinwxd) [reply presto-cassandra/pom.xml:21]: yeah I have tested the version with the community code and it worked fine

**Review by elbinpallimalilibm — APPROVED** [review]

**Pull request merged**